com.chartboost.helium/package.json

```diff
@@ -100,4 +100,4 @@
     "url": "https://www.chartboost.com/helium"
   },
   "hideInEditor": false
-}
+}
```

Comment:
nit: eol

Reply:
This is automatically parsed by Unity when modifying the version, even if we add it, Unity will remove it eventually :P